packages/env/src/index.ts

```diff
@@ -12,5 +12,7 @@ if (env !== 'production') {
     ELASTIC_PASSWORD: 'secret',
     REDIS_PORT: '6379',
     REDIS_HOST: '0.0.0.0',
+    LATITUDE_URL: 'http://localhost:3000',
+    PLUNK_API_KEY: 'pk_00cade00c372874d2755b01915a4892a772da29e3f8aa9cf', // Public api key, safe to share
```

Comment:
This should be generic in case someone want to use the open source project.

Reply:
True